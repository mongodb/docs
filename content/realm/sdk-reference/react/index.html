<!doctype html>
<head>
  <meta http-equiv="refresh" content="0; url=https://mongodb.com/docs/realm-sdks/react/latest" />
</head>
<body>
  <p><a href="https://mongodb.com/docs/realm-sdks/react/latest">Redirect</a></p>
</body>
