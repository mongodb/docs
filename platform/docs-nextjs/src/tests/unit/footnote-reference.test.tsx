import { render } from '@testing-library/react';
import FootnoteReference from '@/components/footnote/footnote-reference';

import mockData from '../data/footnote-reference.test.json';

it('renders correctly', () => {
  const tree = render(<FootnoteReference id={mockData.id} refname={mockData.refname} />);
  expect(tree.asFragment()).toMatchSnapshot();
});
