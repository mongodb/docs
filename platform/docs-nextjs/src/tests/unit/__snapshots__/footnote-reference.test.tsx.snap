// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders correctly 1`] = `
<DocumentFragment>
  <a
    class="footnote-reference header-buffer"
    href="#footnote-1"
    id="ref-1-id1"
  >
    [1]
  </a>
</DocumentFragment>
`;
